iocage_lib/ioc_start.py

```diff
@@ -656,6 +656,12 @@ def __start_jail__(self):
                 _callback=self.callback,
                 silent=self.silent)
 
+        self.set(
```

Comment:
We should make sure to set this very last. Since an exception could happen.

Reply:
I think we should let it remain like this because even if in case an exception is raised wrt rctl or cpuset, we don't stop the jail

Reply:
Fair enough :)